docs/docs/configuration/integration.md

```diff
@@ -265,6 +265,42 @@ http:
           - Authorization
 ```
 
+## Configuring for use with the caddy v2 `forward_auth` directive
```

Comment:
Build is failing because of broken link. Backticks can not be used in headers otherwise the HTML links cannot be properly generated